docs/content/latest/secure/authentication/host-based-authentication.md

```diff
@@ -81,160 +97,161 @@ hostssl    database  user  IP-address  netmask  auth-method  [auth-options]
 hostnossl  database  user  IP-address  netmask  auth-method  [auth-options]
 ```
 
-The meaning of these fields are described below.
+The fields are described below.
```

Comment:
Suggestion:  
rephrase to get rid of "below" (e.g. some flavour of "following")